### PR TITLE
[DDW-771] Add jest and couple of unit tests

### PR DESCRIPTION
**Pull request by danielmain**
This PR adds Jest library and Unit Tests to the new added functions introduced in branch DDW-706

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@danielmain can you please create a JIRA card for this work?

**Comment by nikolaglumac** [comment]
Also, is this WIP or done?

**Comment by danielmain** [comment]
> Also, is this WIP or done?

Still WIP, @nikolaglumac is checking how to setup the dependencies so the build turns green

**Comment by nikolaglumac** [comment]
> > Also, is this WIP or done?
> 
> WIP, @nikolaglumac is checking how to setup the dependencies so the build turns green

Please always try to add the corresponding labels onto the PRs based on their state 🙏 
Thanks!

**Comment by nikolaglumac** [comment]
@DominikGuzei can you please review this one?

**Comment by nikolaglumac** [comment]
@cleverca22 we need your help here...
It seems that yarn2nix is misbehaving again https://hydra.iohk.io/build/7601195/nixlog/1

> An unexpected error occurred: "EACCES: permission denied, unlink '/nix/store/vafkyzg7fxir9dydr38wd4b0xy1v500h-offline/istanbul-lib-report-3.0.0.tgz'".

☝️  this package should be available and works locally...

File: tests/wallets/unit/wallet-utils.spec.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  });
+
+  // This does not apply if
+  //
+  // wallet is NOT delegating
+  // has no tokens and
+  // remain balance after transaction is 0 ADA.
+
+
+  it('<true> in case of: ' +
```
Comment (renanvalentin) [inline tests/wallets/unit/wallet-utils.spec.js:30]: @danielmain you could use template literals to have multi-line:

```
it(`
line 1
line 2
`, () => {...})
```

**Review by renanvalentin — APPROVED** [review]

**Review by DominikGuzei — APPROVED** [review]

**Pull request merged**